lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +17,34 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
+    if text == '':
+        return None
+    if all(symbol.isalpha() is False for symbol in text):
+        return []
+    level_one = text[:]
+    for symbol in level_one:
+        if symbol in ['!', '(', ')', '-', '[', ']', '{', '}', ';', ':', '"', ',', "'", '¡', '¿', '-',
+                      '<', '>', '/', '?', '@', '#', '$', '%', '^', '&', '*', '_', '~', ':', '⟩', '⟨',
+                      '—', '–', '«', '»']:
+            level_one = level_one.replace(symbol, '.')
+    level_two = ''
+    for idx, symbol in enumerate(level_one):
+        if not (idx not in [-1, 0] and symbol == " " and level_one[idx + 1] == "."):
+            level_two += symbol
+    level_three = ''
+    for idx, symbol in enumerate(level_two):
+        if not(symbol == '.' and level_two[idx - 1] == '.'):
```

Comment:
revert not and just reverse the condition

Reply:
done